scripts/core/editor3/components/Editor3.jsx

```diff
@@ -36,8 +36,11 @@ export class Editor3Component extends React.Component {
 
         this.editorRect = {top: 0, left: 0};
         this.readOnly = props.readOnly;
+        this.pageContainer = $('.page-content-container');
```

Comment:
is this selector hardcoded on purpose ? maybe this could be set in the component parameters if we ever plan to reuse this editor somewhere else ?